landing-page/content/common/release-notes.md

```diff
@@ -70,8 +70,25 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
 
 ## 1.1.0 release
 
+Has been released on 28 November 2022. The 1.1.0 release deprecates various pre-1.0.0 methods:
+
+* Puffin statistics have been [added to the Table API](https://github.com/apache/iceberg/pull/4945)
+* Support for [Table scan reporting](https://github.com/apache/iceberg/pull/5268), which enables collection of statistics of the table scans.
+* Added support for Flink 1.16, dropped support for Flink 1.13
+* Dropped support for Apache Spark 3.0
+* [Fixed the SequenceNumber](https://github.com/apache/iceberg/pull/5913) to represent DataSequenceNumber in the ManifestEntry
```

Comment:
This wasn't a fix, it was an extension. We call the existing sequence number the "data sequence number" and added a "file sequence number" to track when files were added, even if the data is older.

Reply:
True, I had a file sequence number in mind but picked the wrong PR.

we can replace it with this.
Core: Add file seq number to ManifestEntry by @aokolnychyi in https://github.com/apache/iceberg/pull/6002

Reply:
Good catch, updated it to 6002